__tests__/api/api.test.js

```diff
@@ -165,6 +165,79 @@ describe("GET /api/articles", () => {
       });
   });
 });
+describe("FEATURE: GET /api/articles (queries)", () => {
+  test("Articles should be filtered by topic=mitch, sorted by author and ordered ascending ", () => {
+    return request(app)
+      .get("/api/articles?topic=mitch&sort_by=author&order=asc")
+      .then(({ body }) => {
+        const { articles } = body;
+        expect(articles).toBeSortedBy("author", { descending: false });
+        articles.forEach((article) => {
+          expect(article).toHaveProperty("topic", "mitch");
+        });
+        expect(articles[0]).toStrictEqual({
+            article_id: 1,
+            title: "Living in the shadow of a great man",
+            comment_count: "11",
+            topic: "mitch",
+            votes: 100,
+            author: "butter_bridge",
+            created_at: "2020-07-09T20:11:00.000Z",
+            article_img_url:
+              "https://images.pexels.com/photos/158651/news-newsletter-newspaper-information-158651.jpeg?w=700&h=700",
+        })
+      });
+  });
+  test("Articles should be filtered by topic=cats, sorted by votes and ordered descending by default ", () => {
+    return request(app)
+      .get("/api/articles?topic=cats&sort_by=votes")
+      .then(({ body }) => {
+        const { articles } = body;
+        expect(articles).toBeSortedBy("votes", { descending: true });
+        articles.forEach((article) => {
+          expect(article).toHaveProperty("topic", "cats");
+        });
+        expect(articles[0]).toStrictEqual({
+          article_id: 5,
+          votes: 0,
+          comment_count: "2",
+          title: "UNCOVERED: catspiracy to bring down democracy",
+          topic: "cats",
+          author: "rogersop",
+          created_at: "2020-08-03T13:14:00.000Z",
+          article_img_url:
+            "https://images.pexels.com/photos/158651/news-newsletter-newspaper-information-158651.jpeg?w=700&h=700",
+        })
+      });
+  });
```

Comment:
Please write another test for a successful response where there is a topic but it doesn't have any articles.